pyproject.toml

```diff
@@ -49,7 +49,8 @@ dependencies = [
     "utm",
     "pyyaml-include",
     "profast",
-    "NREL-rex"
+    "NREL-rex",
+    "turbine_models @ git+https://github.com/NREL/turbine-models.git",
```

Comment:
Tagging a git version will render this project unable to be updated on PyPI, so this will either have to get converted to a real package (if there are funds I can help you and Patrick streamline this process, if helpful) or be a recommended git installation for the user.